components/fieldgroups/experience-event/events/interacted-with-document-in-dialogue.schema.json

```diff
@@ -20,68 +20,83 @@
   "description": "It tracks interactions with documents in dialogues, including document details and dialogue information.",
   "definitions": {
     "interactedwithdocumentindialogue": {
-      "type": "object",
       "properties": {
-        "xdm:dialogueDetails": {
-          "title": "Dialogue Details",
+        "xdm:interaction": {
+          "title": "Interaction",
           "type": "object",
           "properties": {
-            "xdm:dialogueID": {
-              "title": "Dialogue ID",
-              "type": "string",
-              "description": "Unique identifier of the dialogue."
-            },
-            "xdm:dialogueName": {
-              "title": "Dialogue Name",
-              "type": "string",
-              "description": "The name of the dialogue session."
-            },
-            "xdm:pageURL": {
-              "title": "Page URL",
-              "type": "string",
-              "format": "uri",
-              "description": "URL where the dialogue was initiated."
+            "xdm : documentInDialogue": {
```

Comment:
There are extra spaces in the name. Please change from `xdm : documentInDialogue"` to `xdm:documentInDialogue"`